pizza_cutter/des_pizza_cutter/_pizza_cutter.py

```diff
@@ -230,6 +246,12 @@ def _make_epochs_info(object_data, weights, all_se_info):
         data['fname'][i] = se_info['filename']
         data['file_id'][i] = se_info['file_id']
 
+        sl = all_slices[i]
+        data['row_start'][i] = sl.y_start
+        data['col_start'][i] = sl.x_start
+        data['box_size'][i] = sl.box_size
+        data['psf_box_size'][i] = sl.psf_box_size
+
         if se_info['flags'] == 0:
             data['weight'][i] = weights[iused]
             iused += 1
```

Comment:
The use of the running index here feels fragile and requires everything to always get ordered properly.

Reply:
We should return 

1. a list of used slices
2. a list of weights for those slices
3. a list of the slices that were not used

Reply:
you'll also  need to keep the indices of them etc. anyway, feel free to modify

Reply:
The file index is in the source info.